.github/workflows/pages-deploy.yml

```diff
@@ -0,0 +1,49 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy to GitHub Pages
+on:
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets the GITHUB_TOKEN permissions to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  # Single deploy job since we're just deploying
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: "npm"
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
```

Comment:
Actually it looks like we still need to add this build script 
https://vite.dev/guide/static-deploy

Reply:
I think this looks fine, it's following the GitHub Pages example. `npm run build` will run the vite build script and deploy, no?

Reply:
No, there is no build script in the package.json yet. It is missing the following code from the docs
```
{
  "scripts": {
    "build": "vite build",
    "preview": "vite preview"
  }
}
```

Reply:
Nevermind, looks like I completely missed them and they are already included. Disregard.